### PR TITLE
Clean up UI for new design

### PR DESCRIPTION
**Pull request by olilarkin**
This PR:

- replaces the use of a separate style for "inactive" by using iPlug2's IControl::SetDisabled() on param change. This means the UI state will also change when automated/controlled externally, it wouldn't before
- removes old resources
- removes doubled up knobs in c673b561f513f7ac4cb4633192a772868fea9700, replaces with a new control NamKnobControl
- replaces Bitmap switches in c673b561f513f7ac4cb4633192a772868fea9700 with a new NamSlideSwitchControl - the bitmaps here need cleaning up a bit (also did not try on windows or low-DPI screen yet), also layout is somewhat hacked

### CODE REVIEW

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 1]
```diff
@@ -661,6 +619,28 @@ void NeuralAmpModeler::OnUIOpen()
     this->_SetOutputNormalizationDisableState(!this->mNAM->HasLoudness());
 }
 
+void NeuralAmpModeler::OnParamChangeUI(int paramIdx, EParamSource source)
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:622]: ❤️

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 2]
```diff
@@ -349,94 +383,18 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
...
-    pGraphics->AttachControl(new IBKnobRotaterControl(middleKnobArea, knobRotateBitmap, kToneMid), kNoTag, "kToneMid");
-    pGraphics->AttachControl(
-      new IBKnobRotaterControl(trebleKnobArea, knobRotateBitmap, kToneTreble), kNoTag, "kToneTreble");
-    // Output
-    pGraphics->AttachControl(new IVKnobControl(outputKnobArea, kOutputLevel, "", style));
-    pGraphics->AttachControl(
-      new IBKnobRotaterControl(outputKnobArea, knobRotateBitmap, kOutputLevel), kNoTag, "kOutputLevel");
-
-    // Extend the noise gate action function to set the style of its knob
-    auto setNoiseGateKnobStyles = [&, pGraphics, noiseGateControl](IControl* pCaller) {
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:408]: Oh this is fantastic!

**Review by sdatkinson — CHANGES_REQUESTED** [review]
Overall, lots to love 😄 

Two high-level issues: 

1. [Nit] The blue dots in the knobs no longer "unlight" when they are disabled. Aesthetically, I think I prefer what I saw pre-PR but I don't feel strongly about it.
2. [C] The "normalize" switch no longer disables when a model without loudness metadata is loaded.

Unfortunately, (2) will surely generate a lot of questions, so I need it fixed before I can approve.

Pic of what I see on [4c29ffa](https://github.com/sdatkinson/NeuralAmpModelerPlugin/tree/4c29ffa4e404c2e1b5c80ba0fcdd6e198353debe) when loading [this model](https://drive.google.com/file/d/1mfxaiada0eCS4fD69HmTuouYRlNjTd29/view?usp=share_link): 

image

**Comment by olilarkin** [comment]
I believe i have fixed those two things but I don't have an appropriate .nam to test the disabling of the normalization switch. I also removed the ITextControl "Normalization' which didn't fit so nicely with the UI. I think an icon + tooltip might be a good solution there.

**Comment by sdatkinson** [comment]
> I don't have an appropriate .nam to test the disabling of the normalization switch.

[Same model with loudness metadata](https://drive.google.com/file/d/1mz4_NVwTCohpMstynediiIncvRjdDBoy/view?usp=share_link) in case you get to it before me :)

**Review by sdatkinson — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**